### PR TITLE
Converted to typescript (and fixed quotes not being quotes)

### PR DESCRIPTION
**Pull request by FuZzledd**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!doctype html>
-<html lang=”en”>
+<html lang="en">
 <head>
- <meta charset=”utf-8">
+ <meta charset="utf-8">
  <title>My React App from Scratch</title>
 </head>
 <body>
- <div id=”app”></div>
```
Comment (JoshAdamPowell) [inline src/index.html:8]: lol

**Review by JoshAdamPowell — APPROVED** [review]
I think we want yarn.lock to be gitignored (I know this is my fault for not adding it in the first place!). Can you add it to gitignore and remove it from this branch?

**Comment by FuZzledd** [comment]
> I think we want yarn.lock to be gitignored (I know this is my fault for not adding it in the first place!). Can you add it to gitignore and remove it from this branch?

I think it's recommended that you do keep it checked in? See bottom of https://classic.yarnpkg.com/en/docs/yarn-lock/

**Comment by JoshAdamPowell** [comment]
oh cool, I'll just merge this then

**Pull request merged**